trackman/templates/library/labels.html

```diff
@@ -19,8 +19,20 @@ <h1>Library</h1>
     <li class="nav-item"><a class="nav-link" href="{{ url_for('trackman_library.fixup') }}">Fixup</a></li>
 </ul>
 
-<div class="card-body panel-5-columns">
-    <div class="list-group-inline">
+<div class="card-body">
+    <h3>Search</h3>
+    <p>Only searches exact matches</p>
+    <form action="{{ url_for('trackman_library.label') }}" method="get" class="form-horizontal" role="form">
+    <div class="form-group row">
+        <label for="artist" class="col-sm-1 control-label col-form-label">Label</label>
+        <div class="col-sm input-group">
+	    <input class="form-control" type="text" id="artist" name="artist">
```

Comment:
Don't you mean label? Also, please expand the tab and close the tag.

Reply:
copy error. will fix